apps/editing-toolkit/editing-toolkit-plugin/block-patterns/class-block-patterns-from-api.php

```diff
@@ -278,4 +280,24 @@ private function update_pattern_post_types() {
 			}
 		}
 	}
+
+	/**
+	 * Ensure that all query or posts patterns use the category blog instead.
+	 */
+	private function update_query_patterns_with_blog_category() {
```

Comment:
Thanks for thinking about testing 🙌 
What kind of test would be useful for the changes in this PR?

Reply:
![badge](https://img.shields.io/badge/review-imo-orange.svg)

What I was thinking was a test ensuring the structure changes of query patterns from `Posts` to `Blog`. We might be able to [mock the pattern object](https://github.com/Automattic/wp-calypso/blob/trunk/apps/editing-toolkit/editing-toolkit-plugin/block-patterns/test/class-block-patterns-from-api-test.php#L34-L48) with query patterns `Posts` and ensure the changes to `Blog`.
(we can test it through `register_patterns` since `update_query_patterns_with_blog_category` is a private method)

It's gonna be a small test, so I don't mean we need to be hardcore for this.
But it might help prevent future changes break it 💫 , if someone else modifies the code in a way that patterns are incorrectly categorized, the test will fail.

Reply:
Got it. I think for the categories case, we should be more flexible than in other cases because pattern categorization can break easily as Designers can add patterns and categories easily from the admin.

Let's reconsider adding tests later when we know better which areas should not change or things will break and tests will be more helpful.